health/warnings.go

```diff
@@ -0,0 +1,216 @@
+	"fmt"
+)
+
+/**
+This file contains definitions for the Warnables maintained within this `health` package.
+*/
+
+// updateAvailableWarnable is a Warnable that warns the user that an update is available.
+var updateAvailableWarnable = Register(&Warnable{
+	Code:     "update-available",
+	Title:    "Update available",
+	Severity: SeverityMedium,
+	Text: func(args map[string]string) string {
+		return fmt.Sprintf("An update from version %s to %s is available. Run `tailscale update` or `tailscale set --auto-update` to update.", args["CurrentVersion"], args["AvailableVersion"])
+	},
+})
+
+// securityUpdateAvailableWarnable is a Warnable that warns the user that an important security update is available.
+var securityUpdateAvailableWarnable = Register(&Warnable{
+	Code:     "security-update-available",
+	Title:    "Security update available",
+	Severity: SeverityHigh,
+	Text: func(args map[string]string) string {
+		return fmt.Sprintf("An urgent security update from version %s to %s is available. Run `tailscale update` or `tailscale set --auto-update` to update now.", args["CurrentVersion"], args["AvailableVersion"])
+	},
+})
+
+// unstableWarnable is a Warnable that warns the user that they are using an unstable version of Tailscale
+// so they won't be surprised by all the issues that may arise.
+var unstableWarnable = Register(&Warnable{
+	Code:     "is-using-unstable-version",
+	Title:    "Using an unstable version",
+	Severity: SeverityLow,
+	Text: func(args map[string]string) string {
+		return "This is an unstable version of Tailscale meant for testing and development purposes: please report any bugs to Tailscale."
+	},
+})
+
+// NetworkStatusWarnable is a Warnable that warns the user that the network is down.
+var NetworkStatusWarnable = Register(&Warnable{
+	Code:     "network-status",
+	Title:    "Network down",
+	Severity: SeverityHigh,
+	Text: func(args map[string]string) string {
+		return "Tailscale cannot connect because the network is down. (No network interface is up.)"
+	},
+}, WithConnectivityImpact())
+
+// IPNStateWarnable is a Warnable that warns the user that Tailscale is stopped.
+var IPNStateWarnable = Register(&Warnable{
+	Code:     "wantrunning-false",
+	Title:    "Not connected to Tailscale",
+	Severity: SeverityLow,
+	Text: func(args map[string]string) string {
+		return "Tailscale is stopped."
+	},
+})
+
+// localLogWarnable is a Warnable that warns the user that the local log is misconfigured.
+var localLogWarnable = Register(&Warnable{
+	Code:     "local-log-config-error",
+	Title:    "Local log misconfiguration",
+	Severity: SeverityLow,
+	Text: func(args map[string]string) string {
+		return fmt.Sprintf("The local log is misconfigured: %v", args["LocalLogConfigError"])
+	},
+})
+
+// LoginStateWarnable is a Warnable that warns the user that they are logged out,
+// and provides the last login error if available.
+var LoginStateWarnable = Register(&Warnable{
+	Code:     "login-state",
+	Title:    "Logged out",
```

Comment:
I don't think this should be a warnable. We tell the GUI the state already. It can render this however it wants.

Reply:
Agree, happy to get rid of this as it also makes things more messy in the Apple clients. However, isn't health the only way the CLI surfaces its offline/logged out status? If so, we should probably take care of this in a separate PR, otherwise we're going to drop things from the CLI output